implementations/rust/ockam/ockam_core/src/key_exchange.rs

```diff
@@ -21,10 +21,10 @@ extern crate core;
 #[cfg(feature = "alloc")]
 extern crate alloc;
```

Comment:
You should actually delete lines 8 through 22 here, as they're now covered by ockam/src/lib.rs — sorry that I missed this.

This will resolve the issue you have in CI.